From 2f908136e31125fd28004c95c7921e693d3e3673 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Fri, 29 Aug 2014 13:25:22 +0800
Subject: [PATCH 07/26] net: filter: minor: fix kdoc in __sk_run_filter

commit 01d32f6e5a3f709a90aadbb73723e77a96d67cb2 upstream

This minor patch fixes the following warning when doing
a `make htmldocs`:

  DOCPROC Documentation/DocBook/networking.xml
Warning(.../net/core/filter.c:135): No description found for parameter 'insn'
Warning(.../net/core/filter.c:135): Excess function parameter 'fentry' description in '__sk_run_filter'
  HTML    Documentation/DocBook/networking.html

Reported-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Cc: Alexei Starovoitov <ast@plumgrid.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: He Zhe <zhe.he@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 net/core/filter.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/core/filter.c b/net/core/filter.c
index 3733381190ec..765556ba32ef 100644
--- a/net/core/filter.c
+++ b/net/core/filter.c
@@ -125,10 +125,10 @@ noinline u64 __bpf_call_base(u64 r1, u64 r2, u64 r3, u64 r4, u64 r5)
 /**
  *	__sk_run_filter - run a filter on a given context
  *	@ctx: buffer to run the filter on
- *	@fentry: filter to apply
+ *	@insn: filter to apply
  *
  * Decode and apply filter instructions to the skb->data. Return length to
- * keep, 0 for none. @ctx is the data we are operating on, @filter is the
+ * keep, 0 for none. @ctx is the data we are operating on, @insn is the
  * array of filter instructions.
  */
 unsigned int __sk_run_filter(void *ctx, const struct sock_filter_int *insn)
-- 
1.8.1.2

